micro-rdk/src/common/camera/mod.rs

```diff
@@ -1,13 +1,23 @@
-use super::{generic::DoCommand, status::Status};
+use super::{generic::DoCommand, registry::ComponentRegistry, status::Status};
 use bytes::BytesMut;
 use std::sync::{Arc, Mutex};
 use thiserror::Error;
 
-// Enables FakeCamera for native server
-#[cfg(all(feature = "camera", feature = "native", feature = "builtin-components"))]
+#[cfg(feature = "builtin-components")]
 mod fake_camera;
```

Comment:
With the current setup, `FAKE_JPEG` will only be available when both `camera` and `builtin-components` are enabled. otherwise, they're excluded from the module tree during compilation. I made the assumption that as it's not in the tree, it won't be included in the binary.

do you know how to verify this?

Reply:
just look is the symbol gets pulled into the final linked binary. If so we would want to exclude it because it would waste space on flash

Reply:
Lazily and cache or generating on each call?
Would it be alright to file this under a future enhancement ticket?

Reply:
if it's only get linked when camera feature is enabled then i am ok with this.